contracts/axelar-gas-service/src/lib.rs

```diff
@@ -1,9 +1,20 @@
 #![no_std]
 
-mod event;
-mod storage_types;
-
-pub mod contract;
 pub mod error;
 
-pub use contract::AxelarGasServiceClient;
+mod interface;
+
+#[cfg(all(target_family = "wasm", feature = "testutils"))]
+compile_error!("'testutils' feature is not supported on 'wasm' target");
+
+cfg_if::cfg_if! {
+    if #[cfg(all(feature = "library", not(feature = "testutils")))] {
+        pub use interface::{AxelarGasServiceClient, AxelarGasServiceInterface};
+    } else {
+        mod event;
+        mod storage_types;
+
+        pub mod contract;
+        pub use contract::{AxelarGasService, AxelarGasServiceClient};
```

Comment:
are both exports necessary?

Reply:
i'm thinking of keeping only the latter. But want to do that in a separate PR for all contracts